diff --git a/.ci/magma/Makefile b/.ci/magma/Makefile
index 4169aedd03f..9cad921961a 100644
--- a/.ci/magma/Makefile
+++ b/.ci/magma/Makefile
@@ -28,7 +28,7 @@ clean:
 
 .PHONY: magma-cuda130
 magma-cuda130: DESIRED_CUDA := 13.0
-magma-cuda130: CUDA_ARCH_LIST := -gencode arch=compute_80,code=sm_80 -gencode arch=compute_86,code=sm_86 -gencode arch=compute_90,code=sm_90 -gencode arch=compute_100,code=sm_100 -gencode arch=compute_120,code=sm_120
+magma-cuda130: CUDA_ARCH_LIST := -gencode arch=compute_80,code=sm_80 -gencode arch=compute_86,code=sm_86 -gencode arch=compute_90,code=sm_90 -gencode arch=compute_100,code=sm_100 -gencode arch=compute_120,code=sm_120 -gencode arch=compute_121,code=sm_121
 magma-cuda130:
 	$(DOCKER_RUN)
 
